.github/API_REVIEW.md

```diff
@@ -0,0 +1,67 @@
+# API Review
+
+All API changes should go through API review, in addition to our normal code
+review process. We define an API change as
+
+- a change large enough to warrant updating documentation, or
+- a change that increases the maintenance burden of features we offer to our
+  users (i.e., the "API surface area")
+
+For small changes, some or all of these changes can be omitted, but it's best to
+**err on the side of being thorough**. Especially for large changes, you might
+even consider drafting a full-fledged design document.
+
+It's best to go through an API review **before** you start changing the code, so
+that we can offer guidance on how to proceed before getting too caught in the
+weeds.
+
+## Template
+
+Copy/paste this template into a new issue and fill it in to request an API
+review from a maintainer. Remember: depending on the size of your change, it's
+possible to omit some of the sections below.
```

Comment:
Idea: Let's make a tag that we recommend that you add! :) Or some sort of standardized title format?

Reply:
Non-collaborators can't add tags.

I think the fact that it's an API review will be obvious from the template being copy/pasted; how about a collaborator adds the tag?